logging/HTTPOutputHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace SilverStripe\Framework\Logging;
+
+use Monolog\Logger;
+use Monolog\Handler\AbstractProcessingHandler;
+
+/**
+ * Output the error to the browser, with the given HTTP status code.
+ * We recommend that you use a formatter that generates HTML with this.
+ */
+class HTTPOutputHandler extends AbstractProcessingHandler
+{
+
+	private $contentType = "text/html";
+	private $statusCode = 500;
+
+	/**
+	 * Get the mime type to use when displaying this error.
+	 */
+	public function getContentType() {
+		return $this->contentType;
+	}
+
+	/**
+	 * Set the mime type to use when displaying this error.
+	 * Default text/html
+	 */
+	public function setContentType($contentType) {
+		$this->contentType = $contentType;
+	}
+
+	/**
+	 * Get the HTTP status code to use when displaying this error.
+	 */
+	public function getStatusCode() {
+		return $this->statusCode;
+	}
+
+	/**
+	 * Set the HTTP status code to use when displaying this error.
+	 * Default 500
+	 */
+	public function setStatusCode($statusCode) {
+		$this->statusCode = $statusCode;
+	}
+
+	protected function write(array $record) {
+		// TODO: This coupling isn't ideal	
+		if(\Controller::has_curr()) {
```

Comment:
I've run into the same thing a number of times. We should really give this some thought. It would be good to have a github issue number here if we are going to leave a TODO.